### ISSUE TITLE
Document Shortcuts like toggle Menu and Fullscreen in the README 

### ISSUE THREAD

**Issue by phrygian666** [drhelius/Gearsystem#86]
This is very nitpicky, but when switching "Toggle menu off" one can get lost pretty quickly, when not knowing the shortcut. Besides, Gearsystem keeps the settings when restarting (which is good imho) and recompiling. After a while I figured toggle Fullscreen (F11) and later toggle Menu (ctrl + m). I tried to to find some docs online or command line options before that though. Maybe a short section in the README under tips ? 

Also I would reconsider quitting the application with "Esc". Maybe use "Esc" for  Menu focus. I had two times I wanted to save the game but did quit the emulator. This is the Blastem default so I'd blame it on my muscle memory. But I find it too easy to hit. Maybe (ctrl + q)?  

This might be Linux specific. 

Thank you :)

**Comment by drhelius** [comment]
Yes, in the next release I plan to add a little bit of documentation in the Readme.

Thanks for your comments.
